mlrun/serving/server.py

```diff
@@ -771,7 +769,29 @@ async def run(body):
         model_endpoint_uids=model_endpoint_uids,
     )
 
-    return responses
+    # log the results as artifacts
+    num_of_meps_in_the_graph = len(server.graph.model_endpoints_names)
+    artifact_path = mlrun.mlconf.artifact_path
```

Comment:
As we discussed - use the `artifact_path` from the run:
```suggestion
    artifact_path = context.artifact_path
```